### ISSUE TITLE
type error Unknown variable loadMovieNum

### ISSUE THREAD

**Issue by HuengchI** [ASXploit/Actionscript-2-Hack-Loader#1]
I'm using FlashDevelop and got the error when building the project

**Comment by ASXploit** [comment]
I don't believe FlashDevelop compiles ActionScript 2 code. I think it's mainly for ActionScript 3. You'll need a separate compiler aimed toward AS2.

**Comment by HuengchI** [comment]
Thanks. I tried Adobe Flash CS3 and it worked. FlashDevelop uses an open source compiler for AS2 which just doesn't support such magic functions.

**Comment by HuengchI** [comment]
Some games will register their own KEY Event Handlers which override ours. I think it's more stable to trigger our cheats from a Button or ContextMenu.

**Comment by ASXploit** [comment]
If you would like to contribute to an AS2 Hackbar I'm willing to get a project going? Nobody has one out yet.

**Comment by ASXploit** [comment]
I need a team of ActionScript and JavaScript members to revolutionize our hacking community.. They're hard to find and people hide knowledge

**Comment by HuengchI** [comment]
ActionScript 2 is obsoleted so it's extremely hard to find any reference about it. I am currently busy dealing with my college studies and don't have time joining your project. The help doc of Adobe Flash IDE is pretty useful, here's some excerpts:(Creating A Button)

1. Drag the Button component from the components panel into the library of the current document.
This operation adds the component to the library but is not displayed in the application.

2. In frame 1 of the main timeline, add the following ActionScript code to the actions panel to create a Button instance:
`this.createClassObject(mx.controls.Button, "my_button", 10, {label:"Click me"});
my_button.move(20, 20);`

3. Now, add the following ActionScript to create an event listener and an event handler:
`function clicked() {`
    `trace("You clicked the button!");`
`}`
`my_button.addEventListener("click", clicked);`

Best of luck~ :)
